include/volksbot.h

```diff
@@ -59,11 +61,11 @@ class Volksbot
     double wheel_radius_;
     double axis_length_;
     double turning_adaptation_;
-    int gear_ratio_;
     int max_vel_l_;
     int max_vel_r_;
     int max_acc_l_;
     int max_acc_r_;
+    int gear_ratio_;
```

Comment:
Please don't reorder variables without a reason (should give a gcc warning btw.).